app/templates/events/trackVolunteers.html

```diff
@@ -3,10 +3,15 @@
 {% block scripts %}
   {{super()}}
   <script type="module" src="/static/js/trackVolunteers.js"></script>
+  <!-- <script type="module" src="//cdn.datatables.net/1.11.3/js/jquery.dataTables.min.js"></script> -->
+  <script type="text/javascript" src="https://cdn.datatables.net/v/dt/dt-1.11.3/r-2.2.9/datatables.min.js"></script>
```

Comment:
This seems very specific. Do we need to use such a specific version of datatables or can we just use the latest for our jquery version

Reply:
yes, we keep it. This version is for the responsiveness of the data tables.